content/faq/faq.mdx

```diff
@@ -145,6 +147,107 @@ Now default values within the schema file can be used:
   "default": "Max"
 }
 ```
+
+## How to update a field based on another field in JSON Forms?
+
+Consider a carwash application offering various services and calculating the resulting price.
+The schema includes a multiselect field called services and a price attribute, with the price field set as readonly.
+
+<DependentFieldExample/>
+
+There are two approaches to update a field in JSON Forms based on the value of another field:
+
+#### Approach 1: Implementing functionality in the onChange method of JSON Forms
+
+In this approach, you can implement the logic in the onChange method of JSON Forms. 
+The onChange method triggers whenever a user changes the form data.
```

Comment:
Mention that this approach should only be used when using an older JSON Forms without middleware support as it works in a similar way, however the middleware variant is more performant.